postgres_kernel/kernel.py

```diff
@@ -188,10 +188,10 @@ def do_execute(self, code, silent, store_history=True,
             self.send_response(self.iopub_socket, 'stream',
                                {'name': 'stderr', 'text': str(e)})
             self._conn.rollback()
-            return {'status': 'error', 'execution_count': self.execution_count,
-                    'ename': 'ProgrammingError', 'evalue': str(e),
-                    'traceback': []}
+            return {'status': 'abort', 'execution_count': self.execution_count}
+
```

Comment:
Can you talk to me about how this changes the behavior? I'm not sure I see an important difference.

Reply:
With 'error', the workflow is interrupted where the exception is raised. With 'abort' or 'ok' the workflow continues. I chose 'abort' out of assumption that an error should not be 'ok'

Reply:
Hmm, I'm afraid I don't agree that execution should continue. Compare the behavior of the Python kernel: When an exception is raised, no more cells execute.

Maybe I'm misunderstanding?